docs/Dev_eXo_Addons.rst

```diff
@@ -1139,11 +1150,188 @@ following:
+
+Broadcasted events are:
+
+- ``exo.addons.kudos.sent`` : a new Kudos is sent. (Example: `NewKudosSentActivityGeneratorListener <https://github.com/exoplatform/kudos/blob/develop/kudos-services/src/main/java/org/exoplatform/addon/kudos/listener/NewKudosSentActivityGeneratorListener.java>`__)
+- ``exo.addons.kudos.activity`` : a Kudos activity or activity comment is created. (Example: `GamificationIntegrationListener <https://github.com/exoplatform/kudos/blob/develop/kudos-services/src/main/java/org/exoplatform/addon/kudos/listener/GamificationIntegrationListener.java>`__)
+
+To add an event listener using one listed events above, you can add the following configuration inside a  :ref:`Portal extension <PLFDevGuide.eXoAdd-ons.PortalExtension>` configuration file:
+
+   .. code:: xml
+
+      <external-component-plugins>
+        <target-component>org.exoplatform.services.listener.ListenerService</target-component>
+        <component-plugin>
+          <!-- One of the listed event names -->
+          <name>EVENT_NAME</name>
+          <set-method>addListener</set-method>
+          <!-- FQN of the event listener -->
+          <type>org.example.CustomEventListener</type>
+        </component-plugin>
+      </external-component-plugins>
+
+.. _PLFDevGuide.eXoAdd-ons.PerkStoreListeners:
+
+==============================
+eXo Perk Store listeners
+==============================
+
+eXo Perk Store uses :ref:`ListenerService <Kernel.UnderstandingtheListenerService>` to broadcast events about products and orders lifecycle.
+Developers can leverage these events to build custom features or alter external systems.
+Internally, these events are used for a variety of uses such as notifications or updating UI in realtime using Websocket.
+
+Broadcasted events are:
+
+- ``exo.perkstore.settings.modified`` : Perk Store settings gets modified by an administrator. (Example: `WebSocketSettingsListener <https://github.com/exoplatform/perk-store/blob/develop/perk-store-services/src/main/java/org/exoplatform/addon/perkstore/listener/WebSocketSettingsListener.java>`__)
+- ``exo.addons.perkstore.product.createOrModify`` : Perk Store product is created or modified. (Example: `WebSocketProductListener <https://github.com/exoplatform/perk-store/blob/develop/perk-store-services/src/main/java/org/exoplatform/addon/perkstore/listener/WebSocketProductListener.java>`__)
+- ``exo.addons.perkstore.order.createOrModify`` : Perk Store product order is created or modified. (Example: `WebSocketOrderListener <https://github.com/exoplatform/perk-store/blob/develop/perk-store-services/src/main/java/org/exoplatform/addon/perkstore/listener/WebSocketOrderListener.java>`__)
+
+To add an event listener using one listed events above, you can add the following configuration inside a  :ref:`Portal extension <PLFDevGuide.eXoAdd-ons.PortalExtension>` configuration file:
```

Comment:
events=>event

Reply:
In fact, it should be "events", but by adding "of".

> one "of" listed events above

Reply:
done